### DISCUSSION TITLE
Deep-sleep power consumption in v7

### DISCUSSION THREAD

**Discussion by martinberlin** [vroland/epdiy#357] [Ideas]
Comes from #323 Where I made some charts using Nordik PPK and published them in [the WiKi](https://github.com/vroland/epdiy/wiki/Board-v7-Consumption)

@vroland / @mickeprag and any other relevant interested party:
The current v7 consumes about 390 uA (Almost 0.4 mA) in deepsleep. Doing nothing at all and MCU in the most low-consumption mode. This should be ideally below 10 uA.
Tests made feeding the PCB "as a battery" UART chip should be not powered. 

![v7_consumption](https://github.com/user-attachments/assets/154fa81d-3401-4a47-b843-8938775766ce)

I know it's a difficult task since there are a lot of things connected. Any ideas how we can make this lower?

I'm tempted to make a new version using: https://www.sg-micro.com/product/SGM6029 relatively new Buck converted that is used on some Seeed modules and it's proved to be very low consumption, but I'm sure the LDO is not to blame for this. Can you please help me make a more efficient low consumption version?

**Comment by huming2207** [comment]
This level of current looks like a leakage from GPIO. Maybe try tweaking the pullup/down config?

**Comment by vroland** [comment]
That sounds plausible. I don't think I tuned that yet for the S3. @martinberlin do you think you could look into that, since you have the PPK?

**Reply by martinberlin** [reply]
Will try to find time for this as soon as possible. Will try to check state of gpio and make many different tests to see if I find out the root of the problem
